spec/controller_helpers_spec.rb

```diff
@@ -109,7 +109,7 @@ def initialize
         allow(test).to receive(:will_send_request).and_raise(error)
         allow(test).to receive(:response).and_return(response)
 
-        expect(test).to receive(:on_error).with(error, nil, hash_including('mordor.js'))
+        expect(test).to receive(:on_error).with(error, nil, array_including(hash_including({name: "mordor.js"})))
```

Comment:
please use hash rockets (because we support 1.9, but they're also more consistent in general since they're required for non-symbol keys)
```suggestion
        expect(test).to receive(:on_error).with(error, nil, array_including(hash_including({ :name => "mordor.js" })))
```